tests/test_jax_nonbonded.py

```diff
@@ -477,3 +477,15 @@ def reweight(sig_ligand, eps_ligand, q_ligand):
 
         np.testing.assert_allclose(v_ref, v_test)
         np.testing.assert_allclose(gs_ref, gs_test)
+
+
+def test_lj_not_nan():
+    distances = [0.0, 0.00000001, 0.00001, 0.001, np.inf]
```

Comment:
I think we want to be in 32-bit mode when we test this - 64-bit has significantly more precision. Does `0.00000001` result in a `nan` in 64-bit as well?

Reply:
(I thought all the `test_jax_nonbonded.py` files ran in 64bit-jax, but I can't actually find the config line at the top anymore)

Reply:
>I think we want to be in 32-bit mode when we test this - 64-bit has significantly more precision. Does 0.00000001 result in a nan in 64-bit as well?

Ahh, good call -- 0.0 results in a nan in 64-bit, but 0.00000001 does not. 

Updated test to exercise both 32-bit and 64-bit mode: https://github.com/proteneer/timemachine/pull/1085/commits/f0f8bf5a56b0711303208584ecdfe4b98e32af12 ( using https://jax.readthedocs.io/en/latest/_autosummary/jax.experimental.enable_x64.html#jax.experimental.enable_x64 )